### PR TITLE
Return user info instead of id (#10)

### LINKED ISSUE #10: Zwróć dane użytkowników zamiast ich id

### PR DESCRIPTION
**Pull request by jdudycz**

### CODE REVIEW

File: bcycle-backend/Models/Entities/GroupTrip.cs [hunk 1]
```diff
@@ -34,5 +36,22 @@ public void Update(GroupTripRequest data)
...
+                Id = Id,
+                Name = Name,
+                Description = Description,
+                Host = await userProvider(HostId),
+                TripCode = TripCode,
+                StartDate = StartDate,
+                EndDate = EndDate,
+                Route = Route,
+                Participants = Participants
+                    .Select(p => userProvider(p.UserId))
```
Comment (K-Wojciechowski) [inline bcycle-backend/Models/Entities/GroupTrip.cs]: How do I access `ParticipantStatus` now? The UI must know whether a participant is approved or not to display the appropriate controls or to hide them.

**Review by K-Wojciechowski — CHANGES_REQUESTED** [review]

File: bcycle-backend/Models/Entities/GroupTrip.cs (hunk 1 above)
Comment (jdudycz) [reply bcycle-backend/Models/Entities/GroupTrip.cs]: Right, my bad

**Review by K-Wojciechowski — APPROVED** [review]

**Comment by K-Wojciechowski** [comment]
Looks good to me. I resolved a merge conflict from a small API change (enums as strings) I made yesterday to make the Android side happy.

**Comment by jdudycz** [comment]
> Looks good to me. I resolved a merge conflict from a small API change (enums as strings) I made yesterday to make the Android side happy.

Ok, but just leave that kind of things to the author next time. I'll ask if something won't be clear :)

**Pull request merged**